numpy/core/tests/test_simd.py

```diff
@@ -0,0 +1,332 @@
+import unittest
```

Comment:
These should be written as pytest tests, with simple asserts rather than with unittest syntax.

Reply:
I added it into my task list, also left a NOTE in the top file for it.

Reply:
I think there's nothing simpler than assert keyword as @eric-wieser suggested